cli/cppcheckexecutor.cpp

```diff
@@ -311,6 +314,25 @@ namespace {
             return mCtuInfo;
         }
 
+        void readActiveCheckers() {
+            if (mCheckersFile.empty())
+                return;
+
+            std::ifstream fout(mCheckersFile);
+            if (fout.is_open())
+            {
+                std::set<std::string> activeCheckers;
+                std::string line;
+                // cppcheck-suppress accessMoved - FP
+                while (std::getline(fout, line))
+                {
+                    // cppcheck-suppress accessMoved - FP
```

Comment:
This code was copied and the false positives are already being tracked in https://trac.cppcheck.net/ticket/11300.